.cirrus.yml

```diff
@@ -134,7 +134,7 @@ deb_build_task:
         apt -y install devscripts equivs python3-pip
         mk-build-deps
         apt -y install ./sosreport-build-deps*.deb
-        pip3 install avocado-framework==94.0 --break-system-packages
+        pip3 install "avocado-framework<104.0" --break-system-packages
```

Comment:
Shouldn´t we want to explicitly pin it to the 103 LTS, IE:
`avocado-framework==103` ?

Reply:
based on the documentation from avocado-framework, they suggest to use this format to always pick the latest LTS. So, when 103.1 comes out, this should do it. There is no 103, but 103.0. I hope that makes sense

Reply:
Sounds good :+1: